packages/grafana-ui/src/components/Table/TableNG/TableNG.tsx

```diff
@@ -644,6 +646,26 @@ export function mapFrameToDataGrid({
     availableWidth -= COLUMN.EXPANDER_WIDTH;
   }
 
+  // Row background color function
+  let rowBg: Function | undefined = undefined;
+  for (const field of frame.fields) {
+    // eslint-disable-next-line @typescript-eslint/consistent-type-assertions
+    const fieldOptions = field.config.custom as TableFieldOptionsType;
+    const cellOptionsExist = fieldOptions !== undefined && fieldOptions.cellOptions !== undefined;
+
+    if (
+      cellOptionsExist &&
+      fieldOptions.cellOptions.type === TableCellDisplayMode.ColorBackground &&
+      fieldOptions.cellOptions.applyToRow
+    ) {
+      rowBg = (rowIndex: number): CellColors => {
+        const display = field.display!(field.values.get(rowIndex));
```

Comment:
`Array.get()` is deprecated - can we change to use square bracket notation instead?

```suggestion
        const display = field.display!(field.values[rowIndex]);
```